cicd/run_smoke_tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Wrapper script to run the smoke tests locally
+#
+# Expected Env variables: 
+# SIC_API_URL - The URL of the SIC classfication API to run the tests against
+# SA_ID_TOKEN - A valid Google Identity Token generated from your credentials (assuming you're running locally) 
+#
+
+# Example for Sandbox: 
+# export SIC_API_URL=https://sic-classification-vector-store-670504361336.europe-west2.run.app/v1/sic-vector-store
+if [[ -z "${SIC_API_URL}" ]]; then
+    echo Environment variable SIC_API_URL was not set, setting defualt to Sandbox API URL.
+    export SIC_API_URL=https://sic-classification-vector-store-670504361336.europe-west2.run.app/v1/sic-vector-store
```

Comment:
Is there a way we can do this without putting the URL in code. I'd rather avoid advertising it in git if we can.